api/types/swarm/container.go

```diff
@@ -71,4 +71,5 @@ type ContainerSpec struct {
 	Secrets   []*SecretReference  `json:",omitempty"`
 	Configs   []*ConfigReference  `json:",omitempty"`
 	Isolation container.Isolation `json:",omitempty"`
+	Sysctls   map[string]string   `json:",omitempty"`
```

Comment:
Older API versions should ignore this option, so we should probably add a version check in `createService()` and `updateService()`, where this value is reset if it was sent, but the API version that's used is `< 1.39` 🤔. Similar to https://github.com/moby/moby/blob/53683bd8326b988977650337ee43b281d2830076/api/server/router/swarm/cluster_routes.go#L185-L187

Reply:
I'm unclear on why we should ignore this option for service create. Won't CLIs before this version send nothing at all?

Reply:
This was updated 👍